go.mod

```diff
@@ -202,5 +202,5 @@ require (
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.12.0-sdk-v0.46.11
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.18.0-tm-v0.34.27
+	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.19.0-tm-v0.34.27
```

Comment:
note that this bumps celestia-consensus to the version that no longer includes `Blob`s as a separate field in the `Data` struct. This needs to be done in the same PR if we want to avoid breaking everything